src/auth.ts

```
@@ -28,16 +29,144 @@ export const authConfig = {
+              }),
+              db.pendingInviteValidation.delete({ where: { id: pending.id } }),
+            ]);
+          } else {
+            const inviteCode = await db.inviteCode.findUnique({
+              where: { code: pending.code },
+            });
+            if (inviteCode && inviteCode.currentRedemptions < inviteCode.maxRedemptions) {
+              await db.$transaction([
+                db.inviteCode.update({
+                  where: { id: inviteCode.id },
+                  data: { currentRedemptions: { increment: 1 } },
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'data' should be before 'where'.

(sort-keys-fix/sort-keys-fix)